### PR TITLE
add Linux cert script steps

### PR DESCRIPTION
**Pull request by tangowithfoxtrot**
## Objective

Adds instructions for the newly-added `create_linux_certificate.sh` script from #https://github.com/bitwarden/server/pull/2941

### CODE REVIEW

File: docs/getting-started/server/guide.md [hunk 1]
```diff
@@ -268,6 +268,23 @@ Navigate to the `dev` folder in your server repo and perform the following comma
...
 
+### Linux
+
+1.  Generate the certificates and save them to your certificate store:
+
+    ```bash
+    ./create_certificates_linux.sh
+    ```
+
+2.  You will receive output similar to the following. You will need this information for the next
```
[the remark this answers is not in the document]
Comment (tangowithfoxtrot) [reply docs/getting-started/server/guide.md]: I updated the steps for all OS to reflect the more specific language.

**Review by withinfocus — DISMISSED** [review]
Excellent update across all the platforms for consistency!

**Comment by withinfocus** [comment]
Looks like it needs a Prettier fix still.

**Comment by tangowithfoxtrot** [comment]
Ah. That's what I get for using the GitHub editor 🙃 Done!

**Review by withinfocus — APPROVED** [review]

**Pull request merged**